tests/contrib/gunicorn/test_gunicorn.py

```diff
@@ -185,7 +176,6 @@ def test_no_known_errors_occur(tmp_path):
         SETTINGS_GEVENT_DDTRACERUN,
         SETTINGS_GEVENT_DDTRACERUN_MODULE_CLONE,
         SETTINGS_GEVENT_DDTRACERUN_DEBUGMODE_MODULE_CLONE,
-        SETTINGS_GEVENT_SPANAGGREGATOR_NO_RLOCK,
```

Comment:
nit: Should we keep this for now until we completely remove the env var? Or is it pretty safe to get rid of it preemptively?

Reply:
I think it's safe to get rid of. There is no reason why a customer should use no rlock. We added this flag to allow user's to revert back to the previous behavior.